packages/storage-plus/src/indexes.rs

```diff
@@ -204,6 +185,17 @@ where
         self.sub_prefix(K::SubPrefix::new())
             .range(store, min, max, order)
     }
+
+    pub fn keys<'c>(
```

Comment:
Ahh... you added this method. I was wondering how I missed it before. Good call.